pvlib/test/test_midc.py

```diff
@@ -0,0 +1,62 @@
+import inspect
+import os
+
+import pandas as pd
+from pandas.util.testing import network
+import pytest
+
+from pvlib.iotools import midc
+
+
+test_dir = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe())))
+midc_testfile = os.path.join(test_dir, '../data/midc_20181014.txt')
```

Comment:
perhaps we should add a raw data file as well to decouple the format test from the network test